setup-windows-dev-env.ps1

```diff
@@ -69,7 +69,7 @@ switch ($result) {
     choco.exe upgrade nodejs-lts nvs -y
 
     Write-Output "Installing Docker"
-    choco.exe upgrade docker docker-for-windows -y
+    choco.exe upgrade docker-cli docker-for-windows -y
```

Comment:
Apparently `docker-cli` supersedes `docker`, and is the preferred install path on Windows.

Reply:
Good find, thanks!